packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -496,6 +503,18 @@ export class Markets {
       });
     }
 
+    if (params.templateFilter) {
```

Comment:
Could use a switch on `params.templateFilter`.

Reply:
I wanted to only process `templateOnly` or `customOnly` all other values would be ignored.